systers_portal/templates/account/email.html

```diff
@@ -0,0 +1,78 @@
+{% extends "base.html" %}
+{% load account %}
+{% block title %} - Email{% endblock %}
+{% load crispy_forms_tags %}
+{% block content %}
+  <div class="container">
+    <div class="email email-heading">
+      <h2>E-mail Addresses</h2>
+      <hr>
+      {% if user.emailaddress_set.all %}
+          <p id= "email-subheading">The following e-mail addresses are associated with your account:</p>
+          <form action="{% url 'account_email' %}" class="login form-signup" method="post">
+            {% csrf_token %}
+            <fieldset class="blockLabels">
+
+              {% for emailaddress in user.emailaddress_set.all %}
+              <!--<div class="ctrlHolder">-->
+                <div>
+                  <label for="email_radio_{{forloop.counter}}" class="{% if emailaddress.primary %}primary_email{%endif%}">
+
+                    <input id="email_radio_{{forloop.counter}}" type="radio" name="email" {% if emailaddress.primary or user.emailaddress_set.count == 1 %}checked="checked"{%endif %} value="{{emailaddress.email}}"/>
+
+                      {{ emailaddress.email }}
+                      <br>
+                      {% if emailaddress.verified %}
+                          <span id="verified"><b>Verified</b></span>
+                      {% else %}
+                          <span id="unverified"><b>Unverified</b></span>
+                      {% endif %}
+                      
+                      {% if emailaddress.primary %}<span id="primary"><b>Primary</b></span>{% endif %}
+                  </label>
+                </div>
+              {% endfor %}
+
+              <br>
+              <div>
+                    <button class="primaryAction btn btn-lg btn-primary btn-block" type="submit" name="action_primary" >Make Primary</button>
+                    <button class="primaryAction btn btn-lg btn-primary btn-block" type="submit" name="action_send" >Re-send Verification</button>
+                    <button class="primaryAction btn btn-lg btn-primary btn-block" type="submit" name="action_remove" >Remove</button>
+              </div>
+
+            </fieldset>
+          </form>
+
+      {% else %}
+          <p><strong>Warning:</strong>You currently do not have any e-mail address set up. You should really add an e-mail address so you can receive notifications, reset your password, etc.</p>
```

Comment:
It'll help to add Django html specific indentation in .vimrc if you use that, something like ```autocmd Filetype htmldjango setlocal ts=2 sts=2 sw=2 expandtab```.

Also closetag.vim is great!